trillian/ctfe/handlers_test.go

```diff
@@ -108,6 +109,41 @@ type handlerTestInfo struct {
 	c             *LogContext
 }
 
+const certQuotaPrefix = "CERT:"
+
+func quotaUserForCert(c *x509.Certificate) string {
+	return fmt.Sprintf("%s %s", certQuotaPrefix, c.Subject.String())
+}
+
+func quotaUsersForChain(t *testing.T, pem ...string) []string {
```

Comment:
There's a slight chance of confusion here because the "chain" is different from what the mainline code calls a "chain" -- the main code is `[1:]` but this is `[:]`.

Reply:
Good call, changed to `quotaUsersForIssuers`.